x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -261,6 +261,7 @@ export class RulesClient {
   private readonly fieldsToExcludeFromPublicApi: Array<keyof SanitizedRule> = [
     'monitoring',
     'mapped_params',
+    'snoozeEndTime',
```

Comment:
it should be the same pattern that we used in the saved object schema
```suggestion
    'snooze_end_time',
```

Reply:
It's `snoozeEndTime` in the saved object schema. Snake_case isn't used anywhere else in that schema.